tools/IceRpc.Protobuf.Tools/README.md

```diff
@@ -10,27 +10,27 @@ generator is a .NET program and requires .NET 8 or later.
 Once you've added the IceRpc.Protobuf.Tools NuGet package to your project, the Protobuf files of your project are
 automatically compiled into C# files every time you build this project.
 
-The `protoc` compiler contains several built-in generators, including the C# generator, which is responsible for
+The `protoc` compiler contains several built-in generators, including the C# generator which is responsible for
 generating the Protobuf serialization and deserialization code for C#. In addition to these built-in generators,
 `protoc` also executes the external `protoc-gen-icerpc-csharp` generator which is responsible for generating the
-IceRpc + Protobuf integration code.
+IceRPC + Protobuf integration code.
 
 The `protoc` C# generator generates a `.cs` file for each `.proto` file. The generated C# file has the same name
-as the corresponding `.proto` file, but with the name converted to camel case. For example, `greeter.proto` is compiled
+as the corresponding `.proto` file, but with the name converted to Pascal case. For example, `greeter.proto` is compiled
 into `Greeter.cs`.
 
 The `protoc-gen-icerpc-csharp` generator generates a `.IceRpc.cs` file for each `.proto` file. The generated C# file
-has the same name as the corresponding `.proto` file, but with the name converted to camel case. For example,
-`greeter.proto` is compiled into `Greeter.cs`.
+has the same name as the corresponding `.proto` file, but with the name converted to Pascal case. For example,
+`greeter.proto` is compiled into `Greeter.IceRpc.cs`.
 
 [Source code][source] | [Package][package]
 
-## Protobuf files and Protobuf directories
+## Protobuf files and Protobuf import path
```

Comment:
Should this be plural? `import paths`?
Maybe it's correct as-is, I can't tell from the context. But `directories` used to be plural.

Reply:
Path = several directories.